content/en/docs/security/best-practices.md

```diff
@@ -0,0 +1,246 @@
+    However, this may be open for abuse in multi-tenancy environments and therefore is disabled by default.
+  </details>
+  <details>
+    <summary>Audit Procedure</summary>
+    Check Helm Controller's pod YAML for the arguments used at start-up:
+    
+    `kubectl describe pod -n flux-system -l app=helm-controller | grep -B 5 -A 10 Args`
+  </details>
+
+- Ensure controller was not started with `--insecure-kubeconfig-tls=true`.
+  <details>
+    <summary>Rationale</summary>
+    Disable the enforcement of TLS when accessing the API Server of remote clusters.
+    
+    This flag was created to enable scenarios in which non-production clusters need to be accessed via HTTP. Do not disable TLS in production.
+  </details>
+  <details>
+    <summary>Audit Procedure</summary>
+    Check Helm Controller's pod YAML for the arguments used at start-up:
+    
+    `kubectl describe pod -n flux-system -l app=helm-controller | grep -B 5 -A 10 Args`
+  </details>
+
+### Kustomize Controller
+
+#### Start-up flags
+
+- Ensure controller was not started with `--insecure-kubeconfig-exec=true`.
+  <details>
+    <summary>Rationale</summary>
+    KubeConfigs support the execution of a binary command to return the token required to authenticate against a Kubernetes cluster.
+
+    This is very handy for acquiring contextual tokens that are time-bound (e.g. aws-iam-authenticator).  
+    However, this may be open for abuse in multi-tenancy environments and therefore is disabled by default.
+  </details>
+  <details>
+    <summary>Audit Procedure</summary>
+    Check Kustomize Controller's pod YAML for the arguments used at start-up:
+    
+    `kubectl describe pod -n flux-system -l app=kustomize-controller | grep -B 5 -A 10 Args`
+  </details>
+
+- Ensure controller was not started with `--insecure-kubeconfig-tls=true`.
+  <details>
+    <summary>Rationale</summary>
+    Disable the enforcement of TLS when accessing the API Server of remote clusters.
+    
+    This flag was created to enable scenarios in which non-production clusters need to be accessed via HTTP. Do not disable TLS in production.
+  </details>
+  <details>
+    <summary>Audit Procedure</summary>
+    Check Kustomize Controller's pod YAML for the arguments used at start-up:
+    
+    `kubectl describe pod -n flux-system -l app=kustomize-controller | grep -B 5 -A 10 Args`
+  </details>
+
+- Ensure controller was started with `--no-remote-bases=true`.
+  <details>
+    <summary>Rationale</summary>
+    By default the Kustomize controller allows for kustomize overlays to refer to external bases. 
+    This has a performance penalty, as the bases will have to be downloaded on demand during each reconciliation.<br>
+    When using external bases, there can't be any assurances that the externally declared state won't change. In this case, the source lose its hermetic properties. Changes at the external bases 
```

Comment:
This sentence seems to have been cut short

Reply:
This should be corrected now, PTAL.